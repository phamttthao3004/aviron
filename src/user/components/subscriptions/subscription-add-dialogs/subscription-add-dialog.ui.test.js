import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import SubscriptionAddDialogUI from './subscription-add-dialog.ui';

describe('Subscription Add Dialog UI', () => {
    it('Render Subscription Add Dialog UI component', () => {
        const props = { state: undefined, setOpenAdd: undefined };
        const com = shallow(<SubscriptionAddDialogUI {...props} />);
        expect(shallowToJson(com)).toMatchSnapshot();
    });
});
