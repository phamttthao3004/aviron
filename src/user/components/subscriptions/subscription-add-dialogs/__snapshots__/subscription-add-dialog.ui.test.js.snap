// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscription Add Dialog UI Render Subscription Add Dialog UI component 1`] = `
<CmsDialog
  className="dialog-add-edit-group"
  open={false}
>
  <div
    className="cms-dialog-content"
  >
    <div
      className="cms-dialog-title"
    >
      <span>
        ADD MACHINE
      </span>
    </div>
    <div
      className="cms-dialog-main"
    >
      <div
        className="modal-body edit-group-container"
      >
        <p>
          MACHINE ID
        </p>
        <input
          className="form-control"
          placeholder=""
          type="text"
        />
      </div>
    </div>
    <div
      className="cms-dialog-footer"
    >
      <button
        className="btn btn-default"
        name="button"
        onClick={[Function]}
        type="button"
      >
        Cancel
      </button>
      <button
        className="btn btn-primary"
        disabled={false}
        name="button"
        type="button"
      >
        Add
      </button>
    </div>
  </div>
</CmsDialog>
`;
